src/components/multi-select/multi-select.css

```diff
@@ -0,0 +1,86 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+@import "../../styles/typography-v2.css";
+
+.container {
+  composes: tag-list from '../tag-list/tag-list.css';
+  height: auto;
+  margin: var(--spacing-small) 0;
+  padding-top: 0;
+  padding-bottom: 0;
+  position: relative;
+}
+
+.icon-clear {
+  cursor: pointer;
+  padding: var(--spacing-small);
+}
+
+.icon-container {
+  composes: icons from '../form-control-icons/form-control-icons.css';
+  top: calc(var(--spacing-large) + var(--spacing-small));
+}
+
+.input {
+  border: none;
+  flex-grow: 1;
+  flex-shrink: 1;
+  height: var(--spacing-x-large);
+  width: 0;
+}
+
+.input-readonly {
+  composes: input;
+  background-color: transparent;
+}
+
+.form-control-children-container {
+  align-items: center;
+  border: 1px solid var(--mds-grey-12);
+  border-radius: 3px;
+  box-sizing: border-box;
+  font: var(--mds-type-content);
+  height: auto;
+  margin: 0;
+  padding: 0 calc(var(--spacing-x-large) + var(--spacing-medium)) 0 var(--spacing-medium);
+  width: 100%;
+}
+
+.form-control-children-container-invalid {
+  composes: form-control-children-container;
+  border: 1px solid var(--mds-red-100);
+}
+
+.form-control-children-container-readonly {
+  composes: form-control-children-container;
+  background-color: var(--mds-grey-3);
+}
+
+.popup-container {
+  composes: container from '../listbox/listbox.css';
+  background-color: var(--white);
+  box-sizing: border-box;
+  max-height: 160px;
+  overflow-y: scroll;
+  position: absolute;
+  width: 100%;
+  z-index: 10;
+}
+
+.no-options {
+  composes: popup-container;
+  composes: no-options from '../no-options/no-options.css';
+  color: var(--mds-grey-54);
+  display: flex;
+  justify-content: center;
+  overflow-y: unset;
+}
+
+.tag-list {
+  composes: tag-list from '../tag-list/tag-list.css';
```

Comment:
Why do we have two tag-lists in this component? I think this might have been an oversight?

Reply:
I think you're referring to this `compose` appearing twice. If so, you're correct, the instance in `.container` was extraneous now after layout changes earlier in the dev cycle, and has been removed.